.github/fixtures/npx_scripts.json

```diff
@@ -0,0 +1,75 @@
+{
+	"$schema": "./fixtures.schema.json",
+	"fixtures": [
+		{
+			"checkout": false,
+			"name": "cli help",
+			"script": [
+				"npx --verbose @google-labs/breadboard-cli --help"
+			]
+		},
+		{
+			"checkout": false,
+			"name": "cli run",
+			"script": [
+				"wget https://raw.githubusercontent.com/ExaDev-io/breadboard-samples/develop/samples/Breadboard%20Walkthrough/src/hello-world/board.json -O board.json",
+				"npx --verbose @google-labs/breadboard-cli run board.json"
+			]
+		},
+		{
+			"checkout": true,
+			"name": "npm run test:pack",
+			"script": [
+				"npm install",
+				"pushd packages/breadboard-cli",
+				"npm run test:pack"
+			]
+		},
+		{
+			"checkout": true,
+			"name": "pack && cli run",
+			"script": [
+				"npm install",
+				"pushd packages/breadboard-cli",
+				"npm run pack:install",
+				"cd ~ # this cd is required so the workspace's node_modules aren't available",
+				"wget https://raw.githubusercontent.com/ExaDev-io/breadboard-samples/develop/samples/Breadboard%20Walkthrough/src/hello-world/board.json -O board.json",
+				"npx --verbose @google-labs/breadboard-cli run board.json"
+			]
+		},
+		{
+			"checkout": true,
+			"name": "pack && cli mermaid",
+			"script": [
+				"npm install",
+				"cd packages/breadboard-cli",
+				"npm run pack:install",
+				"cd ~ # this cd is required so the workspace's node_modules aren't available",
+				"wget https://raw.githubusercontent.com/ExaDev-io/breadboard-samples/develop/samples/Breadboard%20Walkthrough/src/hello-world/board.json -O board.json",
```

Comment:
Yeah, I wanted to in the first place. However, I couldn't immediately see any boards in the main repo that would run without any inputs.

Reply:
re-using the schema test board as it's a similar purpose. rather than poluting the `.github` folder
https://github.com/breadboard-ai/breadboard/pull/789/files#diff-c9934dd1c52cf6bfdf97098df88e5fc07000127ac45b10be598ebf382b4ce59bR17